src/components/Functions/ServoCalibration.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+    import {writable} from "svelte/store";
+    import {openSidecar} from "../../utils/sidecar";
+    import ToastStore from "../../stores/toast";
+    import Select from "../Input/Select.svelte";
+    import SuspenseButton from "../Input/Button/SuspenseButton.svelte";
+
+    const state = writable({
+            form: {
+                calibrationTimer: 5,
+                rightHand: true,
+            },
+            loading: false,
+    });
+
+    const beginCalibration = async (extend) => {
```

Comment:
"begin calibration" doesn't quite describe what's going on with the servos, should have different name than the copy/paste from auto calibrate

Reply:
This was from before we decided that the button were to be used for testing (for calibrating) rather than just calibrating. The switch came after I wrote this and forgot to change :P Resolved in later commit.